ci/release/update-version.sh

```diff
@@ -37,7 +37,7 @@ function sed_runner() {
     sed -i.bak ''"$1"'' $2 && rm -f ${2}.bak
 }
 
-sed_runner "s/set(RAPIDS_VERSION .*)/set(RAPIDS_VERSION \"${NEXT_SHORT_TAG}\")/g" cpp/examples/cmake/thirdparty/fetch_rapids.cmake
+sed_runner "s/set(RAPIDS_VERSION .*)/set(RAPIDS_VERSION \"${NEXT_SHORT_TAG}\")/g" examples/cpp/cmake/thirdparty/fetch_rapids.cmake
```

Comment:
I don't think this change is necessary: https://github.com/rapidsai/cuvs/blob/branch-24.06/examples/cmake/thirdparty/fetch_rapids.cmake

Reply:
Fixed @raydouglass